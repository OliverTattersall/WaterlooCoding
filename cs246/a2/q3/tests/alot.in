a 0 r 1 1 4 4
a 0 g 0 0 1 1
a 0 b 0 0 1 2
a 0 b 1 1 4 4
a 0 b 1 0 3 4
a 0 b 0 1 3 4
a 0 y 5 5 1 1
a 0 o 1 1 2 2
a 0 a 2 4 2 4
a 0 w 0 0 3 3
a 0 r 1 1 4 4
a 0 g 0 0 1 1
a 0 b 0 0 1 2
a 0 b 1 1 4 4
a 0 b 1 0 3 4
a 0 b 0 1 3 4
a 0 y 5 5 1 1
a 0 o 1 1 2 2
a 0 a 2 4 2 4
a 0 w 0 0 3 3
pc0
