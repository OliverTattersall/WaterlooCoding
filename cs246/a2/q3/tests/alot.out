Dimensions: 6x6
Number of rectangles: 20
	[Red (1,1) 4x4]
	[Green (0,0) 1x1]
	[Blue (0,0) 1x2]
	[Blue (1,1) 4x4]
	[Blue (1,0) 3x4]
	[Blue (0,1) 3x4]
	[Yellow (5,5) 1x1]
	[Orange (1,1) 2x2]
	[Black (2,4) 2x4]
	[White (0,0) 3x3]
	[Red (1,1) 4x4]
	[Green (0,0) 1x1]
	[Blue (0,0) 1x2]
	[Blue (1,1) 4x4]
	[Blue (1,0) 3x4]
	[Blue (0,1) 3x4]
	[Yellow (5,5) 1x1]
	[Orange (1,1) 2x2]
	[Black (2,4) 2x4]
	[White (0,0) 3x3]
